dotcom-rendering/src/web/server/articleToHtml.tsx

```diff
@@ -99,7 +99,10 @@ export const articleToHtml = ({ article }: Props): string => {
 	 * Please talk to the dotcom platform team before adding more.
 	 * Scripts will be executed in the order they appear in this array
 	 */
-	const loadCommercial = !article.isAdFreeUser && !article.shouldHideAds;
+	const loadCommercial =
+		!article.isAdFreeUser &&
+		!article.shouldHideAds &&
+		article.config.abTests.poorDeviceConnectivityVariant !== 'variant';
```

Comment:
Are you 100% sure that’s the name of the test? Hard to test before it’s out, I guess… we can always fix-forward.

Reply:
based on this? https://github.com/guardian/frontend/blob/2717b8daa4c41295df7c072b1cd533725e624b44/common/app/experiments/Experiments.scala#L77-L86